### PR TITLE
Style ui login

### PR DESCRIPTION
**Pull request by danggrianto**
I love reef-pi, however the UI part need to be polished. I am not a UI expert but I can help clean up a bit.

### CODE REVIEW

**Comment by ranjib** [comment]
can you rebase against master?

**Comment by ranjib** [comment]
also fix the lint errors, https://travis-ci.org/reef-pi/reef-pi/builds/385470127

File: jsx/sign_in.jsx [hunk 1]
```diff
@@ -43,25 +82,14 @@ export default class SignIn extends React.Component {
...
-            <div className='col-sm-2'>Password</div>
-            <div className='col-sm-2'><input type='password' id='reef-pi-pass' /></div>
-          </div>
-          <div className='row modal-footer'>
-            <div className='col-sm-2' />
-            <div className='col-sm-2'><input type='button' role='confirm' ref='confirm' value='sign-in' onClick={this.saveCreds} id='btnSaveCreds' /></div>
-          </div>
+      <div className='container' style={outerStyle}>
+        <div className='form' style={formStyle}>
+          <h1 className='h3 mb-3 font-weight-normal'>Reef Pi</h1>
```
Comment (ranjib) [inline jsx/sign_in.jsx]: reef-pi, all small case and with a hyphen :l-)

**Comment by ranjib** [comment]
@danggrianto can you update the reef-pi spelling from "Reef Pi" to "reef-pi" ? Looks good to me otherwise
thanks

**Comment by danggrianto** [comment]
@ranjib done

**Pull request merged**